apps/src/templates/sectionsRefresh/SectionsSetUpContainer.jsx

```diff
@@ -272,6 +273,12 @@ export default function SectionsSetUpContainer({
   };
 
   const renderAdvancedSettings = () => {
+    // TODO: this will probably eventually be a setting on the course similar to hasTextToSpeech
+    // currently we're working towards piloting in Javalab in CSA only.
+    const aiTutorAvailable =
+      experiments.isEnabled('ai-tutor-toggle') &&
```

Comment:
nit: Do we have a convention on camelcase vs. snake case for experiments? The Blockly experiment is `enableModalFunctionEditor`.

Reply:
I think there may be some inconsistency on casing because some experiments are frontend only and some are also backend, and backend ones tend not to use camel case. since this is going to be a backend experiment, dashes make sense to me, but I'm certainly open to a different policy.

Reply:
There's a mix of camelCase and kabob-case (and even under_scores 😱) in [experiments.js](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/util/experiments.js). I opted for kabob since it's more common and consistent with the `ai-rubrics` experiment, which seems most analogous to what we're doing here. I'm on board with a backend/frontend distinction for consistency if folks feel strongly.